packages/tocco-ui/src/EditableValue/typeEditors/Duration.js

```diff
@@ -0,0 +1,143 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import SPAN from '../../Typography/Typography'
+
+import styled from 'styled-components'
+const StyledDuration = styled.div`
```

Comment:
Separate styles from functionality by files. Move declaration of StyledDuration from ./typeEditors/Duration.js to ./typeEditors/StyledDuration.js.